examples/agentic/dummy_adaptive.py

```
@@ -0,0 +1,139 @@
+    Returns:
+        True if quality has degraded, False otherwise
+    """
+    score_change = current_score - previous_score
+    percent_change = (score_change / previous_score * 100) if previous_score != 0 else 0
+    
+    if percent_change > 2:
+        trend = "improving"
+    elif percent_change < -2:
+        trend = "degrading"
+    else:
+        trend = "stable"
```

Ruff (0.12.2)

Local variable `trend` is assigned to but never used

Remove assignment to unused variable `trend`

(F841)